pydoctor/astbuilder.py

```diff
@@ -40,6 +47,12 @@ def _maybeAttribute(cls: model.Class, name: str) -> bool:
     obj = cls.find(name)
     return obj is None or isinstance(obj, model.Attribute)
 
+class SkipInlineDocstring(Exception):
+    ...
+class InvalidName(Exception):
```

Comment:
This is a better name for this exception:

```suggestion
class NotAName(Exception):
```